### PR TITLE
JAMES-1991 Gatling + live metrics propaganda

### PR DESCRIPTION
**Pull request by chibenwa**
I also corrected a **.gitignore** fail during previous rebase.

### CODE REVIEW

**Comment by chibenwa** [comment]
Link for online visualisation https://rawgit.com/chibenwa/james-project/JAMES-1991-show/site/server/metrics.html

**Comment by chibenwa** [comment]
screenshot from 2017-04-11 11-04-20

screenshot from 2017-04-11 11-04-25

screenshot from 2017-04-11 11-04-27

screenshot from 2017-04-11 11-04-49

screenshot from 2017-04-11 11-05-10

Some screen shots...

File: src/site/xdoc/server/feature-performance.xml [hunk 1]
```diff
@@ -27,23 +27,76 @@
...
+      <img src="images/performances/metrics.png"/>
+
+      <ul>We track so far:
+         <li>Protocols detailed time execution (and counts)</li>
+         <li>Percentile of mailet and matcher executions</li>
+         <li>Enqueue and Dequeue time</li>
+      </ul>
+
+      <ul>All these solutions allow us to identify the components that need improvement. Here are some numbers of
+        Linagora's James instalation:
```
Comment (mbaechler) [inline src/site/xdoc/server/feature-performance.xml]: you should not talk about Linagora here

File: src/site/xdoc/server/feature-performance.xml [hunk 2]
```diff
@@ -27,23 +27,76 @@
...
+         <li>Around 15.000 incoming e-mails a day</li>
+         <li>42.500 IMAP commands a day</li>
+         <li>20.000 JMAP requests a day</li>
+      </ul>
+
+      <p>More details on metrics are available on <a href="metrics.html">the metric page</a>.</p>
+    </subsection>
+
+    <subsection name="Measuring dequeueing speed">
+      <p>James can spool more than 100 mails per second (configured with Maildir mail storage).</p>
```
Comment (mbaechler) [inline src/site/xdoc/server/feature-performance.xml]: is it still true ?

Comment (chibenwa) [inline src/site/xdoc/server/feature-performance.xml]: I guess with maildir backend. We did not tested. I did not touch. Do you want to remove the entire section?

File: src/site/xdoc/server/feature-performance.xml (hunk 1 above)
Comment (chibenwa) [inline src/site/xdoc/server/feature-performance.xml]: Ok

**Comment by chibenwa** [comment]
Updated. Thanks @mbaechler for review

File: src/site/xdoc/server/feature-performance.xml [hunk 3]
```diff
@@ -78,26 +78,6 @@
       <p>More details on metrics are available on <a href="metrics.html">the metric page</a>.</p>
     </subsection>
 
-    <subsection name="Measuring dequeueing speed">
```
Comment (rouazana) [inline src/site/xdoc/server/feature-performance.xml]: I'd prefer we keep this for now, and check that is it still right as soon as possible.

**Pull request merged**